test/boost/cdc_test.cc

```diff
@@ -307,6 +316,8 @@ SEASTAR_THREAD_TEST_CASE(test_primary_key_logging) {
         assert_row(1, 11, 444);
         // UPDATE ks.tbl SET val = 555 WHERE pk = 2 AND pk2 = 11 AND ck = 111 AND ck2 = 1111
         assert_row(2, 11, 111, 1111);
+        // UPDATE ks.tbl USING TTL 3600 SET val = 444 WHERE pk = 3 AND pk2 = 11 AND ck = 111 AND ck2 = 1111
+        assert_row(3, 11, 111, 1111, 3600);
```

Comment:
hm, do I misremember how row markers and ttls work? I was under the impression that this kind of `UPDATE` will only update the TTL for changed columns, without touching the row marker (which will only be set if data is inserted with `INSERT`). If that's the case, shouldn't `ttl` field in the cdc table remain untouched here? I saw in the `transformer` code that this column is only responsible for keeping the row ttl, i.e. the one kept in the row marker, leaving per-cell ttls to be kept in `res` (whatever it's used for).
It's very likely that there's no bug here at all and I just misunderstand how this code is supposed to work, but in any case, please clarify for the sake of my education :)

Reply:
We would like to use the `ttl` column for TTLs that are set both by INSERTs and UPDATEs. If you follow the logic closely in `transformer::transform` you will see that `row_ttl` will contain either ttl of the row marker, or from a cell. I see now that `row_ttl` is a bad name and why it is confusing, so I changed it to a more neutral `ttl`.

Reply:
👌